pkg/azdiskschedulerextender/azdiskschedulerextender_metrics.go

```diff
@@ -14,13 +16,14 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package main
+package azdiskschedulerextender
```

Comment:
I just noticed this change, we actually need a main package in the extender.

Reply:
nvm, I thought this was changed in all the files :D

Reply:
You are right. I'll revert this change.